docs/diagrams/ModelClassDiagram.puml

```diff
@@ -40,17 +44,21 @@ ModelManager o--> "1" AddressBook
 ModelManager o-left-> "1" UserPrefs
 UserPrefs .up.|> ReadOnlyUserPrefs
 
-AddressBook *--> "1" UniquePersonList
-UniquePersonList o--> "*" Person
-Person *--> Name
-Person *--> Phone
-Person *--> Email
-Person *--> Address
-Person *--> "*" Tag
+AddressBook *--> "1" UniqueStudentList
+UniqueStudentList o--> "*" Student
+Student *--> Name
+Student *--> MatriculationNumber
+Student *--> Faculty
+Student *--> Phone
+Student *--> Email
+Student *--> Address
+Student *--> VaccinationStatus
+Student *--> MedicalDetails
+Student *--> "*" SchoolResidence
```

Comment:
Correct me if I'm wrong, does this mean a student can have 0 or many school residences? Shouldn't it be only 1 school residence for every student?

Reply:
Amended